documents/tree.go

```diff
@@ -105,6 +105,14 @@ func (cd *CoreDocument) DocumentSaltsFunc() func(compact []byte) ([]byte, error)
 	}
 }
 
+// FingerprintSaltsFunc returns an empty function
+func (cd *CoreDocument) FingerprintSaltsFunc() func(compact []byte) ([]byte, error) {
+	return func(compact []byte) ([]byte, error) {
+		randbytes := make([]byte, 32)
+		return randbytes, nil
```

Comment:
this function is should always return the same salt preferably nil bytes else it wouldn't be different from the CodeDocument salts function